Documentation/DocuBlocks/Rest/Database/get_api_database_new.md

```diff
@@ -19,6 +19,11 @@ Default replication factor for new collections created in this database.
 Special values include "satellite", which will replicate the collection to
 every DB-server, and 1, which disables replication.
 
+@RESTSTRUCT{minReplicationFactor,get_api_database_new_USERS,string,optional,number}
+Default minimum replication factor for new collections created in this database.
+You can not have less replication.
```

Comment:
I think a brief description of what the "minimum replication factor" actually does and what side effects it can cause would be super helpful here.